incubator/nodejs-loopback/stack.yaml

```diff
@@ -1,7 +1,10 @@
 id: nodejs-loopback
-name: LoopBack 4 application
+name: LoopBack 4 Framework for Node.js
```

Comment:
nit: LoopBack 4 Node.js Framework?

I'm for keeping the name as short as possible. take a look at the current stacks on https://appsody.dev/